content/installation/net/docker/Installation-docker.md

```diff
@@ -0,0 +1,41 @@
+<!--
+title: "Installing Contrast .NET Framework agent in Docker"
+description: "Guide to installing .NET agent on Docker for Windows containers using the NuGet package"
+tags: "installation configuration .net docker nuget visual studio"
+-->
+
+The following instructions show you how install and configure the Contrast .NET agent in Docker for Windows using the NuGet package. Examples of the finished code exist in a [GitHub repository](https://github.com/Contrast-Security-OSS/contrast-dotnet-examples).
```

Comment:
@ldemidov Did I describe this accurately?

Reply:
yep